azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -260,6 +260,13 @@ public boolean accept(File dir, String name) {
     return allProjects;
   }
 
+  private boolean isActive() throws ExecutorManagerException {
```

Comment:
it will be used in FlowRunnerManager's cleanup thread. Cleanup thread won't perform deletion when isActive returns false